kroxylicious-api/src/main/java/io/kroxylicious/proxy/plugin/package-info.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/**
```

Comment:
@SamBarker @robobario @gracegrimwood following our discussion I wrote this PTAL.